tests/manage/test_pvc_deletion_during_io.py

```diff
@@ -50,3 +52,15 @@ def test_run_io_and_delete_pvc(self):
         except exceptions.CommandFailed as ex:
             if "NotFound" in str(ex):
                 pass
+
+    @tier1
```

Comment:
we should keep consistency with tiers decorators. decorating test classes / decorating test methods

Reply:
it can be different for different scenarios.
you can have a single class with 2 different functions from a different tier levels